cli/core/docs/cli/style_guide.md

```diff
@@ -193,7 +193,7 @@ $ tanzu apps workload delete <name> --wait-timeout 10m
 
 Available for plugins written in golang.
 
-CLI commands should utilize the plugin component library in [pkg/cli/component](https://github.com/vmware-tanzu/tanzu-framework/tree/main/pkg/v1/cli/component) for interactive features like prompts or table printing.
+CLI commands should utilize the plugin component library in [cli/runtime/component](https://github.com/vmware-tanzu/tanzu-framework/tree/main/cli/runtime/component) for interactive features like prompts or table printing.
```

Comment:
Nice job fixing the links 👍 

I found another one at line 134 that needs ajusting

Reply:
Thank you. Fixed.